libs/Zend/Db/Adapter/Mysqli.php

```diff
@@ -299,6 +299,15 @@ protected function _connect()
 
         $this->_connection = mysqli_init();
 
+        $use_ssl = false;
+        $ssl_options = array (
+            1012 => null, // 1012 = PDO::MYSQL_ATTR_SSL_CA
+            1013 => null, // 1013 = PDO::MYSQL_ATTR_SSL_CAPATH
+            1011 => null, // 1011 = PDO::MYSQL_ATTR_SSL_CERT
+            1014 => null, // 1014 = PDO::MYSQL_ATTR_SSL_CIPHER
+            1010 => null, // 1015 = PDO::MYSQL_ATTR_SSL_KEY
```

Comment:
* the comment says `1015` but code says `1010` 
 * maybe we could use these const directly instead of numeric values (in case they later change) eg. `PDO::MYSQL_ATTR_SSL_CIPHER` (if they are defined/available)